src/main/java/task/Task.java

```diff
@@ -0,0 +1,54 @@
+package task;
+/**
+ * Task is the superclass of the activities that can be stored within core.Duke.
+ */
+public abstract class Task {
+    private boolean isComplete;
+    private final String taskDescription;
+
+    /**
+     * Constructs a Task with the provided name and status.
+     *
+     * @param description The description or name of the task
+     * @param isComplete The status of this task
+     */
+    protected Task(String description, Boolean isComplete) {
+        this.taskDescription = description;
+        this.isComplete = isComplete;
+    }
+
+    /**
+     * Returns the name of the Task.
+     * @return The description or name of this Task.
+     */
+    public String getTaskDescription() {
```

Comment:
Use of appropriate naming conventions for class methods in your task package. Names also follow camelCase, e.g. toCsv() and not toCSV(), which is a common mistake. Names of boolean sound like booleans.👍

Reply:
Thank you for your kind words!